examples/sunlinsol/lapackdense/test_sunlinsol_lapackdense.c

```diff
@@ -132,13 +132,12 @@ int main(int argc, char *argv[])
   /* Print result */
   if (fails) {
     printf("FAIL: SUNLinSol module failed %i tests \n \n", fails);
-    printf("\nA (original) =\n");
-    SUNDenseMatrix_Print(B,stdout);
```

Comment:
Was printing of the dense matrices causing the test failures, or were these removed for some other reason?

Reply:
It was not contributing to the failures, but it was making deciphering the log difficult.